dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23CDIGeneralTests.java

```diff
@@ -218,6 +218,8 @@ public void testCDIManagedProperty() throws Exception {
         String contextRoot = "CDIManagedProperty";
         try (WebClient webClient = new WebClient()) {
             webClient.setAjaxController(new NicelyResynchronizingAjaxController());
+            // Avoid error: message=[missing ; after for-loop condition]
+            webClient.getOptions().setThrowExceptionOnScriptError(false);
```

Comment:
Would you mind opening an issue and linking here so we can check if we can remove this once we pull in the new MyFaces javascript?

Reply:
Otherwise, these changes look good. Thanks!

Reply:
I've created https://github.com/OpenLiberty/open-liberty/issues/23135